config/settings.yml

```
@@ -0,0 +1,4 @@
+max_length:
+  email: 255
```

thay vì đưa tên field thế này thì ko tái sử dụng dc, đổi thành 
```suggestion
length:
  gitit_255: 255
```